pennylane/measurements/measurements.py

```diff
@@ -681,6 +681,22 @@ def process_state(self, state: Sequence[complex], wire_order: Wires):
                 dimension :math:`2^n` acts on a subspace of :math:`n` wires
         """
 
+    def process_density_matrix(self, density_matrix: Sequence[complex], wire_order: Wires):
+        """
+        Process the given density matrix.
+
+        Args:
+            density_matrix (Sequence[complex]): The density matrix representing the (mixed) quantum state,
+                which may be single or batched. For a single matrix, the shape should be `(2**n, 2**n)`
```

Comment:
```suggestion
                which may be single or batched. For a single matrix, the shape should be ``(2**n, 2**n)``
```

rst and sphinx are double back ticks for code blocks. comment applies to the other code locations too.

Reply:
Thanks for pointing this out! Just corrected other locations